packages/manager/modules/cloud-connect/src/cloud-connect.component.js

```diff
@@ -0,0 +1,9 @@
+import template from './template.html';
+
+export default {
+  bindings: {
+    cloudConnect: '<',
+    cloudConnectId: '<',
```

Comment:
It seems redundant to bind both `cloudConnect` and `cloudConnectId`. Can't we get the `cloudConnectId` from the `cloudConnect` ?